ResourceStatusSystem/Utilities/Utils.py

```diff
@@ -37,43 +39,44 @@ def getCSTree( csPath = '' ):
 
   opHelper = Operations()
 
-  def getCSTreeAsDict( treePath ):
+  def getCSTreeAsDict(treePath):
     '''
       Function to recursively iterate over a CS tree
     '''
 
     csTreeDict = {}
 
-    opts = opHelper.getOptionsDict( treePath )
-    if opts[ 'OK' ]:
+    opts = opHelper.getOptionsDict(treePath)
+    if opts['OK']:
 
-      opts = opts[ 'Value' ]
+      opts = opts['Value']
 
-      for optKey, optValue in opts.items():
-        if optValue.find( ',' ) > -1:
-          optValue = List.fromChar( optValue )
+      for optKey, optValue in opts.iteritems():
```

Comment:
Wouldn't python-modernize just undo these changes (items -> iteritems) again?

Reply:
python-modernize on this one (with my modification) will do:
```
--- ResourceStatusSystem/Utilities/Utils.py	(original)
+++ ResourceStatusSystem/Utilities/Utils.py	(refactored)
@@ -6,11 +6,13 @@
 
 '''
 
+from __future__ import absolute_import
 import fnmatch
 
 from DIRAC import gConfig, S_OK
 from DIRAC.Core.Utilities import List
 from DIRAC.ConfigurationSystem.Client.Helpers.Operations import Operations
+import six
 
 __RCSID__ = '$Id:  $'
 
@@ -51,7 +53,7 @@
 
       opts = opts['Value']
 
-      for optKey, optValue in opts.iteritems():
+      for optKey, optValue in six.iteritems(opts):
         if optValue.find(',') > -1:
           optValue = List.fromChar(optValue)
         else:
```
So, python-modernize won't undo the changes. But, If I restore what was before (`.items()` again):
```
--- ResourceStatusSystem/Utilities/Utils.py	(original)
+++ ResourceStatusSystem/Utilities/Utils.py	(refactored)
@@ -6,6 +6,7 @@
 
 '''
 
+from __future__ import absolute_import
 import fnmatch
 
 from DIRAC import gConfig, S_OK
```

That line won't be touched. Which, to me, it's very surprising, because I was expected it to be changed in 
`for optKey, optValue in list(opts.items())`
(notice the `list()` that I added).

Reply:
Thanks, interesting about the items(), cf. https://github.com/DIRACGrid/DIRAC/pull/3765/files?utf8=%E2%9C%93&diff=split&w=1#r205694440

Reply:
I do not understand why it does `list(opts.items())`, and not `list(opts)`. It is the same no, just a double copy ?